### PR TITLE
[9.0][IMP] product_pricelist_direct_print: Add order option

### PR DESCRIPTION
**Pull request by carlosdauden**
@Tecnativa

### CODE REVIEW

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Please add Spanish translation as well

File: product_pricelist_direct_print/views/report_product_pricelist.xml [hunk 1]
```diff
@@ -58,6 +58,14 @@
...
                                t-value="products and products.with_context(categ_id=category.id).filtered(lambda x: x.categ_id.id == x.env.context['categ_id']) or
                                         products.search([('sale_ok','=',True), ('categ_id','=',category.id)])"/>
+                            <t t-if="o.order_field == 'name'">
+                                <t t-set="category_products"
+                                   t-value="category_products.sorted(lambda x: x.name)"/>
+                            </t>
+                            <t t-if="o.order_field == 'default_code'">
+                                <t t-set="category_products"
+                                   t-value="category_products.sorted(lambda x: x.default_code)"/>
+                            </t>
```
Comment (rafaelbn) [inline product_pricelist_direct_print/views/report_product_pricelist.xml:68]: Minor style remark

**Review by rafaelbn — APPROVED** [review]
Tested functionally in runbot 👍 Thanks!

**Review by luismontalba — APPROVED** [review]
Tested on runbot

**Comment by carlosdauden** [comment]
@rafaelbn Changes done

**Comment by carlosdauden** [comment]
@pedrobaeza Changes done

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**